modules/mod-deep-learning/ModelCard.cpp

```diff
@@ -304,7 +308,14 @@ void ModelCard::Deserialize(const Doc& doc, const Doc& schema)
 
    // these first three fields are not in HF metadata but rather added later,
    // so don't throw if they are not present (empty default values are given)
-   m_name              = tryGetString( "name",              doc, false).value_or("");
+   
+   m_modelname              = tryGetString( "name",              doc, false).value_or("");
+   std::optional<std::string> displayname = tryGetString( "displayname",              doc, false);
+   if (displayname)
+      m_displayname = displayname.value();
+   else
+      m_displayname = m_modelname;
+
```

Comment:
Minor comment. The model developer will have to use the keywords "name" and "displayname". Why not change "name" to "modelname" ? Just for consistency.